cli/cli/.goreleaser.yml

```diff
@@ -10,11 +10,17 @@ builds:
     goos:
       - darwin
       - linux
+      - windows
     goarch:
       - amd64
       - arm64
     env:
-      - CGO_ENABLED=1
+      - >-
+          {{- if eq .Os "windows" }}
+            CGO_ENABLED=0
+          {{- else }}
+            CGO_ENABLED=1
```

Comment:
we need CGO_ENABLED For the LSP to work :(

Reply:
Added more information on the PR